Processing_data/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.IO;
+using Newtonsoft.Json;
+using Processing_data.Classes;
+
+string json = File.ReadAllText("data.json");
+DataValue[] data = JsonConvert.DeserializeObject<DataValue[]>(json);
+
+if (data == null)
+{
+    Console.WriteLine("Failed to deserialize the JSON data");
+    return;
+}
+PrintingStatsService printingService = new PrintingStatsService();
+StatsBucket statsBucket = new StatsBucket(printingService);
+
+foreach (DataValue dataValue in data)
```

Comment:
we need 5 buckets in the end, each containing 100 values.

Reply:
add Capacity 5 for Dictionary